packages/@ourworldindata/grapher/src/fullScreen/FullScreen.tsx

```diff
@@ -12,10 +13,13 @@ export class FullScreen extends React.Component<{
     content: React.RefObject<HTMLDivElement> = React.createRef()
 
     @action.bound onDocumentClick(e: React.MouseEvent): void {
+        const settingsDrawer = document.getElementById(GRAPHER_DRAWER_ID)
```

Comment:
nit: `settingsDrawer` is maybe not the right name any more?

Reply:
yap :)